packages/stark-ui/src/modules/pagination/pagination.module.ts

```diff
@@ -8,7 +8,7 @@ import { MatTooltipModule } from "@angular/material/tooltip";
 import { MatPaginatorModule } from "@angular/material/paginator";
 import { StarkPaginationComponent } from "./components";
 import { StarkSvgViewBoxModule } from "../svg-view-box/svg-view-box.module";
-import { StarkKeyboardDirectivesModule } from "../keyboard-directives/keyboard-directives.module";
+import { StarkRestrictInputDirectiveModule } from "../restrict-input-directive/restrict-input-directives.module";
```

Comment:
Can you put the name of the module in singular since the module has only 1 directive? In this case it would be: `restrict-input-directive.module.ts`

Reply:
👍 Good catch. 😄